### DISCUSSION TITLE
package centric install for non-developers

### DISCUSSION THREAD

**Discussion by lbdyck** [zopencommunity/meta#345] [Ideas]
most sites will not be developing and will only need the production ready packages - there should be an easy way to install via zopen, or ??, just the desired package/packages - obviously with dependencies. This install should be to a user defined target filesystem and a usable `.profile`  created that can be incorporated into the `/etc/profile` for all to use. 

the install could be into `/u/user/tmp/` and then be remounted at `/usr/` so the generated `.profile` would have to account for that.

**Comment by IgorTodorovskiIBM** [comment]
Hey @lbdyck , have you tried Russell's latest zopen fork? More info here: https://github.com/DevonianTeuchter/meta/blob/main/README.md
This should cover most, if not all, of your pain points:
https://github.com/DevonianTeuchter/meta/releases/latest
We're hoping to get it merged in soon, but if you have any early feedback, that would be great!

**Reply by lbdyck** [reply]
thanks - i'll review at some point - i really don't have time right now to start from scratch :)

**Comment by MikeFultonDev** [comment]
@lbdyck have you had a chance to try out our significant re-work of 'zopen'? I hope this addresses your needs. If not, please comment!
